ui.frontend/src/view/FormOptionFieldBase.js

```diff
@@ -62,6 +63,18 @@ class FormOptionFieldBase extends FormFieldBase {
         }
     }
 
+    updateValidityForRadioButtonAndCheckbox(validity) {
```

Comment:
Please override in respective components, let's not pollute field base with component specific details

Reply:
Implemented!